content/client-server-api/modules/push.md

```diff
@@ -156,7 +156,7 @@ The different `kind`s of rule, in the order that they are checked, are:
     These configure behaviour for (unencrypted) messages that match certain
     patterns. Content rules take one parameter: `pattern`, that gives the
     glob pattern to match against. This is treated in the same way as
-    `pattern` for `event_match`.
+    `pattern` for `event_match` against a `key` of `content.body`.
```

Comment:
I mean it's true (I think), and certainly an improvement over what was there before, but it seems a very complicated way of defining it, particularly given the behaviour of `content.body` is itself defined as a special-case of the more general `event_match` behaviour.

Could we define the behaviour from first principles rather than by reference to `event_match`, including the definition of a word boundary and the behaviour of `content.body` is absent or not a string? Possibly the definition of a `glob` should be factored out to a separate section after "Conditions" that can be linked to where we mention globs.

Let me know what you think: if it's a pain in the butt that you don't have time for then we can land this as-is as an incremental improvement.

Reply:
> Possibly the definition of a "glob-style pattern" should be factored out to a separate section after "Conditions" that can be linked to whenever we mention globs.

or even to an appendix so that (now, or in the future) it can be referred to from https://pr1441--matrix-spec-previews.netlify.app/client-server-api/#events-19 and https://pr1441--matrix-spec-previews.netlify.app/client-server-api/#mroomserver_acl

Reply:
I can give it a go. 👍 I didn't realize the globbing was used somewhere else, an appendix sounds reasonable.

Reply:
I've attempted this -- the appendix section is rather short, but it is nice to have the same reference from a bunch of places. Let me know what you think!